src/ra_utility.h

```diff
@@ -103,6 +103,9 @@ filesize(std::basic_string<CharT>&& filename) noexcept
     return file.tellg();
 } // end function filesize
 
+typedef const TCHAR *LPCTSTR;
+typedef TCHAR *LPTSTR;
+
```

Comment:
Not a big deal but these should be `usings `just in case the compiler complains that the type signature doesn't match.

Reply:
these were copied line-for-line from the windows header they were defined in so I could include this file without having to include the windows header. will try with the using.